vep/__init__.py

```diff
@@ -45,19 +46,17 @@ def _power_on_self_test(self):
         double-check that required parts are in place.
         :return: bool
         """
-        # path to python must end match python$ or you will confuse virtualenv-tools
-        if not re.search('python$', self.args.python):
-            raise ConfigurationError('The path to python must match \'python$\', you tried to use: {0}'.format(self.args.python))
-        # path to python has to be a link/real file, not a symlink.
-        python_real_path = os.path.realpath(self.args.python)
-        if python_real_path == self.args.python:
+        # path to python has to be a link/real file, not a symlink; if it is a
+        # symlink, follow it and use that.
+        python_real_path = os.path.realpath(self.python)
+        if python_real_path == self.python:
             pass
         else:
-            raise ConfigurationError(
-                "The python specified isn't a proper file. Python we tried to use is: {0}; the real path to that is: {1}".format(
-                    self.args.python, python_real_path
+            self.logger.info("you asked to use{0}, which is a link to {1}, so we are using that.".format(
+                    self.python, python_real_path
```

Comment:
Yes, but "I'm doing something other than what you expect" seems like a warning.

Not blocking merge, but that's my stance still.